documentation/asciidoc/accessories/camera/filters.adoc

```diff
@@ -32,40 +32,39 @@ image::images/hoyacm500.png[CM500 Transmission Graph,width="65%"]
 
 == IR Filter
 
-Both the High Quality Camera and Global Shutter Camera contain an IR filter, which is used to reduce the camera's sensitivity to infrared light. This ensures that outdoor photos look more natural. However, some nature photography can be enhanced with the removal of this filter; the colours of sky, plants, and water can be affected by its removal. The camera can also be used without the filter for night vision in a location that is illuminated with infrared light.
+Both the High Quality Camera and Global Shutter Camera contain an IR filter to reduce the camera's sensitivity to infrared light and help outdoor photos look more natural. However, you may remove the filter to:
 
-=== Filter Removal
-
-The filter removal procedure applies to both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
+* Enhance colours in certain types of photography, such as images of plants, water, and the sky
+* Provide night vision in a location that is illuminated with infrared light
 
-WARNING: Before proceeding read through all of the steps and decide whether you are willing to void your warranty. *Do not proceed* unless you are sure that you are willing to void your warranty.
+=== Filter Removal
 
 WARNING: *This procedure cannot be reversed:* the adhesive that attaches the filter will not survive being lifted and replaced, and while the IR filter is about 1.1mm thick, it may crack when it is removed. *Removing it will void the warranty on the product*. Nevertheless, removing the filter will be desirable to some users.
 
+You can remove the filter from both the HQ and GS cameras. The HQ camera is shown in the demonstration below.
+
 image:images/FILTER_ON_small.jpg[width="65%"]
 
-Work in a clean and dust-free environment, as the sensor will be exposed to the air. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away.
+NOTE: Make sure to work in a clean and dust-free environment, as the sensor will be exposed to the air. 
 
+1. Unscrew the two 1.5 mm hex lock keys on the underside of the main circuit board. Be careful not to let the washers roll away.
```

Comment:
Instead of writing step numbers manually, prefer just a dot (`.`). Asciidoctor still recognises this as an ordered list, and will automatically generate the right numbers for you.

This way, you won't end up with cascading multi-line changes in a set of steps if you remove or add a step.

Reply:
If this isn't in the style guide already, it should be :-)